relatio/narrative_models.py

```diff
@@ -327,6 +323,12 @@ def fit_static_clustering(
             # Grid search
             models = []
             scores = []
+            plot_args = {}
```

Comment:
These code snippets clutter the readability of the code. Could all this be part of the plotting function instead of the fit function?